src/coreclr/tools/aot/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -256,7 +256,8 @@
     <Compile Include="IBC\ReaderExtensions.cs" />
     <Compile Include="Compiler\ProfileData.cs" />
     <Compile Include="Compiler\ProfileDataManager.cs" />
-    <Compile Include="Compiler\ReadyToRunHashCode.cs" />
+    <Compile Include="..\..\..\System.Private.CoreLib\src\Internal\VersionResilientHashCode.cs" />
```

Comment:
Having multiple files with the same name does not work particularly well in the IDEs. I would use different file names for the partial slices.

Also, I would place the common part under $(LibrariesProjectRoot)\Common. There is a prior art for including files from under $(LibrariesProjectRoot)\Common directory in the AOT compilers. There is no prior art for including files from CoreLib in the AOT compilers.

Reply:
Done.